src/include/platform/internal/GenericPlatformManagerImpl.h

```diff
@@ -57,6 +57,8 @@ class GenericPlatformManagerImpl
     void _RemoveEventHandler(PlatformManager::EventHandlerFunct handler, intptr_t arg);
     void _HandleServerStarted();
     void _HandleServerShuttingDown();
+    void _HandleSoftwareFault(uint32_t eventId);
```

Comment:
Why do we need this API on platform manager?

If we're just trying to queue IM events, shouldn't we have public APIs for that that applications can use without having to go through Server _and_ platform manager?

Reply:
That is the legacy design and we can move everything to related cluster server